CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 1.4.1 (Nov 20, 2023)
```

Comment:
Extra change I believe

Reply:
This is the change log for official release version, it's a cherry-pick from v1.4.1 
we will release that bug fix int he next release I guess